src/firecracker/src/main.rs

```diff
@@ -96,9 +96,11 @@ fn main() -> ExitCode {
     if let Err(err) = result {
         error!("{err}");
         eprintln!("Error: {err:?}");
-        ExitCode::from(err)
+        let exit_code = FcExitCode::from(err) as u8;
+        error!("Firecracker exited with error. exit_code={exit_code}");
```

Comment:
Nice. I would say it would be a bit more correct saying "exiting with error ..." here and above, since the process is still running.